.github/workflows/tests.yml

```diff
@@ -101,41 +101,70 @@ jobs:
           npm install -g nodemon
           npm init -y
           npm install --force
-          npm test -- --coverage || true
+          npm test
       - name: Archive code coverage results
         uses: actions/upload-artifact@v3
         with:
           name: web-app-client
           path: telematic_system/telematic_apps/web_app/client/coverage/*
           if-no-files-found: error
+  web_app_server_coverage:
+    runs-on: ubuntu-latest
+    container:
+      image: node:16.16.0
+      options: --user root
+      env:
+        CI: true
+    steps:
+      - name: Checkout ${{ github.event.repository.name }}
+        uses: actions/checkout@v3
+      - name: Build
+        run: |
+          cd $GITHUB_WORKSPACE/telematic_system/telematic_apps/web_app/server
+          npm init -y
+          npm install --force
+          npm run test --coverage || true
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: web-app-server
+          path: telematic_system/telematic_apps/web_app/server/coverage/*
+          if-no-files-found: error
   sonar:
-    needs: [carma_vehicle_bridge_coverage, carma_cloud_bridge_coverage, telematic_cloud_messaging_coverage, web_app_client_coverage]
```

Comment:
If we are removing these modules from coverage can we add a comment here that they need to be added back?

Reply:
added